combinetf2/physicsmodels/helpers.py

```diff
@@ -0,0 +1,144 @@
+import importlib
+import pkgutil
+import re
+
+import tensorflow as tf
+
+
+def check_and_import(module_name, package_name):
+    full_module_name = f"{package_name}.{module_name}"
+    print(f"Attempting to import module: {full_module_name}")  # Debug print
+
+    try:
+        # Try to import the module
+        module = importlib.import_module(full_module_name)
+        return module
+    except ModuleNotFoundError:
+        print(f"Module '{full_module_name}' not found.")
+        print("Available modules:")
+        # List all available modules under the package
+        available_modules = [
+            name
+            for _, name, _ in pkgutil.iter_modules(
+                importlib.import_module(package_name).__path__
+            )
+            if name != "helpers"
+        ]
+        for mod in available_modules:
+            print(f"- {mod}")
+        return None
+
+
+def instance_from_module(module_name, *args, **kwargs):
+    class_name = module_name.capitalize()
```

Comment:
I would just make the user specify the class name directly with capitalization as-is.

ie provide directly "Project" on the command line as appropriate

Reply:
OK!